### ISSUE TITLE
retry

### ISSUE THREAD

**Issue by Axel3026** [MehdiRtal/MicrosoftRewardsBot#65]
I am having a problem, when it ends with an error, the other one starts and in the end I do not complete anything is halfway and does not make any mission what can it be? The accounts are like this

Secrets:
```ejemplo@hotmail.com:123456 ejemplo2@hotmail.com:123456```

The Retry works with the second account but the first account is without farmear and the second one only gives me 55 points

![image](https://user-images.githubusercontent.com/87623383/204808276-d20b74bf-ab5a-4365-952e-a2cea1ff0f77.png)

![image](https://user-images.githubusercontent.com/87623383/204808452-cc3d0e8d-bc43-4fe0-88f4-9b9d6404123f.png)

**Comment by Axel3026** [comment]
```
Run timeout 1800 python MicrosoftRewardsBot.py --accounts *** --headless --privacy
[INFO] everyday : None
[INFO] headless : True

[INFO] session : False
[INFO] error : False
[INFO] fast : False
[INFO] privacy : True
[INFO] emailalerts : False
[INFO] redeem : False

[LOGS] "logs.txt" file not found.
[LOGS] "logs.txt" created.

********************EJEMPLO1@hotmail.com********************
[WDM] - Downloading:   0%|          | 0.00/7.26M [00:00<?, ?B/s]
[WDM] - Downloading: 100%|██████████| 7.26M/7.26M [00:00<00:00, 132MB/s]
Error: Process completed with exit code 124.
```
[LOGS] "logs.txt" file not found.
[LOGS] "logs.txt" created.
I think that this can cause errors in log.txt. I'm not sure but it could be that.

**Comment by DerMuffin** [comment]
Got the same error. Retry also failed with the same error Code for me

**Comment by Axel3026** [comment]
@DerMuffin And do you have any idea how to disable the log.txt so that it can be scanned again?

**Comment by Axel3026** [comment]
@b3zd1vpm I want you to help us, since you don't get those kinds of errors, why and how do you do it?

**Comment by dmarts05** [comment]
> @DerMuffin And do you have any idea how to disable the log.txt so that it can be scanned again?

You colud put "rm logs.txt" in the cron task alongside the execution of the script.

**Comment by Axel3026** [comment]
I already discovered that setting it to 1800 caused the error to cut it before, so I increased it to 3600 to test tomorrow.
```
run: timeout 3600 python MicrosoftRewardsBot.py --accounts ${{ secrets.ACCOUNTS }} --headless --fast --privacy
      - name: Retry
        if: ${{ failure() }}
        run: timeout 1800 python MicrosoftRewardsBot.py --accounts ${{ secrets.ACCOUNTS }} --headless --fast --privacy
```

**Comment by Axel3026** [comment]
![image](https://user-images.githubusercontent.com/87623383/205099638-7b57be4a-a863-42b5-9785-6e1aa4308acc.png)
@dmarts05 Here?

**Comment by Axel3026** [comment]
![image](https://user-images.githubusercontent.com/87623383/205135487-907145ed-8e41-44c9-bc24-460ebf3c8ae1.png)
The problem is solved by increasing the time, I finish it in 34 minutes, so to be sure I put it in 1 hour  @DerMuffin

**Comment by DerMuffin** [comment]
@Axel3026 Yes. Because you run more that one Account. I think the problem here is that 3600 Seconds are for all Accounts in the script. If you have more accounts this limit will be exceeded.

**Comment by Axel3026** [comment]
@DerMuffin Yes, but I already did the test, I realized that two accounts with --fast take approximately between 32/33 total minutes, it would be 16 minutes each account.

if you have them like this in the secrets:
```examples1@hotmail.com:pass examples2@hotmail.com:pass```
and the ```--fast``` argument is on takes 33 minutes total. So I could have a timeout of 40 to 45 minutes.

if you don't use the ```--fast``` argument then it takes about 29 minutes per account. It took me about 57 minutes to complete everything.

![image](https://user-images.githubusercontent.com/87623383/205302980-6a348bf4-4985-4d97-9f86-ef63859e896f.png)

If you have more than 2 accounts per deploy like me, we would have to calculate approximately ```29 minutes per account``` if you don't use the ```--fast``` argument. and if you use the ```--fast``` argument. ```17 minutes per account``` and increase the TIMEOUT depending on what you need.

**Comment by MehdiRtal** [comment]
Maybe have timeout * accounts count ?

**Comment by Axel3026** [comment]
@MehdiRtal Yes, you could put at least ```20 minutes per account with --fast``` and without ```--fast 35``` per account to have a safety margin.

**Comment by MehdiRtal** [comment]
I wasn't talking about the "--fast" argument. I'm actually referring to "timeout" that we just added to the workflow.

**Comment by Axel3026** [comment]
I understand, but what if someone doesn't use the argument, I would keep the ```--fast``` setting and still may get errors
```Example 1: It has an express closing in 15 minutes.```
```Example 2: Countdown without express ends in 30 minutes.```
So, if we put a count counter that - ```default timeout do we use?```, we'll put it so that it doesn't cause errors for this argument. if we set a ```Default timeout of 15 minutes```, then people who don't use the ```--fast``` argument will kill sooner and throw errors. I don't know if I explained myself well, sorry for my English
